custom/icds_reports/tests/agg_tests/reports/test_awc_reports.py

```diff
@@ -370,329 +370,8 @@ def test_awc_reports_pse_images_0(self):
-                }
-            ]
-        )
-
-    def test_awc_reports_pse_images_6(self):
-        data = get_awc_reports_pse(
-            {
-                'state_id': 'st1',
-                'district_id': 'd1',
-                'block_id': 'b1',
-                'awc_id': 'a1',
-                'aggregation_level': 5
-            },
-            (2017, 5, 1),
-            'icds-cas'
-        )
-        for kpi in data['kpi']:
-            for el in kpi:
-                del el['help_text']
-        self.assertEqual(
-            data['images'][6],
-            [
-                {
-                    "date": "25/05/2017",
-                    "image": "http://localhost:8000/a/icds-cas/icds_dashboard/icds_image_accessor/"
-                             "20e4d641-a85a-4927-96ab-994fa46a8ea0/1495690578649.jpg",
-                    "id": 24
-                },
-                {
-                    "date": "26/05/2017",
-                    "image": "http://localhost:8000/a/icds-cas/icds_dashboard/icds_image_accessor/"
-                             "f86e701b-1531-469f-8996-705e297bf498/1495776461721.jpg",
-                    "id": 25
-                },
-                {
-                    "date": "27/05/2017",
-                    "image": "http://localhost:8000/a/icds-cas/icds_dashboard/icds_image_accessor/"
-                             "6701b39d-4b6f-4ae3-8a88-eadb61b1a105/1495865744995.jpg",
-                    "id": 26
-                },
-                {
-                    "date": "28/05/2017",
-                    "image": None,
-                    "id": 27
-                }
-            ]
-        )
-
-    def test_awc_reports_pse_images_7(self):
-        data = get_awc_reports_pse(
-            {
-                'state_id': 'st1',
-                'district_id': 'd1',
-                'block_id': 'b1',
-                'awc_id': 'a1',
-                'aggregation_level': 5
-            },
-            (2017, 5, 1),
-            'icds-cas'
-        )
-        for kpi in data['kpi']:
-            for el in kpi:
-                del el['help_text']
-        self.assertEqual(
-            data['images'][7],
-            [
-                {
-                    "date": "29/05/2017",
-                    "image": "http://localhost:8000/a/icds-cas/icds_dashboard/icds_image_accessor/"
-                             "6376d77d-bb2a-48ac-9042-7892dda97bba/1496036503892.jpg",
-                    "id": 28
-                },
-                {
-                    "date": "30/05/2017",
-                    "image": "http://localhost:8000/a/icds-cas/icds_dashboard/icds_image_accessor/"
-                             "c0d002ca-f7b0-4bd2-a531-881b46610c2f/1496120210768.jpg",
-                    "id": 29
-                },
-                {
-                    "date": "31/05/2017",
-                    "image": None,
-                    "id": 30
-                }
-            ]
+            data['images'],
+            []
```

Comment:
good to add one test where we can ensure that it is returning the images. You might need to use mock dates for where you are finding the 90 days back variable.